testing/kuttl/e2e-other/exporter-upgrade/00--cluster.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: exporter
+spec:
+  postgresVersion: 14
+  image: us.gcr.io/container-suite/crunchy-postgres:ubi8-14.0-5.0.3-0
+  instances:
+    - name: instance1
+      dataVolumeClaimSpec:
+        accessModes:
+        - "ReadWriteOnce"
+        resources:
+          requests:
+            storage: 1Gi
+  backups:
+    pgbackrest:
+      repos:
+      - name: repo1
+        volume:
+          volumeClaimSpec:
+            accessModes:
+            - "ReadWriteOnce"
+            resources:
+              requests:
+                storage: 1Gi
+  monitoring:
+    pgmonitor:
```

Comment:
Should we be defining the exporter image explicitly here too :question:

Reply:
That's an interesting question... we are looking for a particular interaction here (between the exporter image and two different pgnodemx versions in two different pg images), and as long as we're hard-coding one image, yeah, I think we might as well.